auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -12,13 +12,8 @@
           <span class="subtitle">{{ $t('myBusinessDashSubtitle') }}</span>
         </h1>
         <div class="view-header__actions">
-          <v-btn
-            id="add-name-request-btn"
-            class="font-weight-regular"
-            color="primary"
-            outlined dark large
-            @click="goToNameRequest()"
-          >
```

Comment:
This is an acceptable code style -- no need to change it.

Also, note that the `>` is in line with the `<` so that elements under this line explicitly look nested.